### ISSUE TITLE
Accuracy.

### ISSUE THREAD

**Issue by mrinal18** [danielegrattarola/twitter-sentiment-cnn#9]
Hey, sorry to bother again, 
What is the accuracy that you are getting when running this? 
i am gettin accuracy of 53% and want to increase it. 
any suggestion?

**Comment by mrinal18** [comment]
Epoch: 7.038456719825784e-05 - loss: 133.074 - acc: 0.51:   0%| | 1/42622.979999Epoch: 0.00014076913439651568 - loss: 132.773 - acc: 0.51:   0%| | 2/42622.97999Epoch: 0.00021115370159477352 - loss: 119.407 - acc: 0.46:   0%| | 3/42622.97999Epoch: 0.00028153826879303136 - loss: 106.877 - acc: 0.49:   0%| | 4/42622.97999Epoch: 0.0003519228359912892 - loss: 122.571 - acc: 0.57:   0%| | 5/42622.979999Epoch: 0.00042230740318954704 - loss: 105.927 - acc: 0.51:   0%| | 6/42622.97999Epoch: 0.0004926919703878049 - loss: 104.487 - acc: 0.48:   0%| | 7/42622.979999Epoch: 0.0005630765375860627 - loss: 109.863 - acc: 0.52:   0%| | 8/42622.979999Epoch: 0.0006334611047843205 - loss: 92.069 - acc: 0.54:   0%| | 9/42622.9799999Epoch: 0.0007038456719825784 - loss: 101.371 - acc: 0.49:   0%| | 10/42622.97999Epoch: 0.0007742302391808363 - loss: 89.7704 - acc: 0.54:   0%| | 11/42622.97999Epoch: 0.0008446148063790941 - loss: 98.222 - acc: 0.5:   0%| | 12/42622.9799999Epoch: 0.000914999373577352 - loss: 100.883 - acc: 0.46:   0%| | 13/42622.979999Epoch: 0.0009853839407756098 - loss: 108.835 - acc: 0.43:   0%| | 14/42622.97999Epoch: 0.0010557685079738677 - loss: 92.1828 - acc: 0.5:   0%| | 15/42622.979999Epoch: 0.0011261530751721254 - loss: 110.24 - acc: 0.46:   0%| | 16/42622.979999Epoch: 0.0011965376423703834 - loss: 104.441 - acc: 0.45:   0%| | 17/42622.97999Epoch: 0.001266922209568641 - loss: 88.6981 - acc: 0.49:   0%| | 18/42622.979999Epoch: 0.001337306776766899 - loss: 79.5906 - acc: 0.57:   0%| | 19/42622.979999Epoch: 0.001407691343965157 - loss: 88.9727 - acc: 0.55:   0%| | 20/42622.979999Epoch: 0.0014780759111634146 - loss: 90.0057 - acc: 0.59:   0%| | 21/42622.97999Epoch: 0.0015484604783616725 - loss: 79.8577 - acc: 0.59:   0%| | 22/42622.97999Epoch: 0.0016188450455599304 - loss: 99.5641 - acc: 0.48:   0%| | 23/42622.97999Epoch: 0.0016892296127581881 - loss: 79.9584 - acc: 0.54:   0%| | 24/42622.97999Epoch: 0.001759614179956446 - loss: 97.0341 - acc: 0.53:   0%| | 25/42622.979999Epoch: 0.001829998747154704 - loss: 101.906 - acc: 0.49:   0%| | 26/42622.979999Epoch: 0.0019003833143529617 - loss: 96.9762 - acc: 0.5:   0%| | 27/42622.979999Epoch: 0.0019707678815512196 - loss: 93.5484 - acc: 0.49:   0%| | 28/42622.97999Epoch: 0.0020411524487494773 - loss: 95.9533 - acc: 0.5:   0%| | 29/42622.979999Epoch: 0.0021115370159477355 - loss: 93.7128 - acc: 0.53:   0%| | 30/42622.97999Epoch: 0.002181921583145993 - loss: 104.124 - acc: 0.43:   0%| | 31/42622.979999Epoch: 0.002252306150344251 - loss: 87.6976 - acc: 0.48:   0%| | 32/42622.979999Epoch: 0.0023226907175425086 - loss: 92.6553 - acc: 0.57:   0%| | 33/42622.97999Epoch: 0.0023930752847407667 - loss: 94.8157 - acc: 0.5:   0%| | 34/42622.979999Epoch: 0.0024634598519390244 - loss: 92.3689 - acc: 0.55:   0%| | 35/42622.97999Epoch: 0.002533844419137282 - loss: 91.964 - acc: 0.52:   0%| | 36/42622.9799999Epoch: 0.0026042289863355403 - loss: 72.7318 - acc: 0.62:   0%| | 37/42622.97999Epoch: 0.002674613553533798 - loss: 83.443 - acc: 0.57:   0%| | 38/42622.9799999Epoch: 0.0027449981207320557 - loss: 105.578 - acc: 0.43:   0%| | 39/42622.97999Epoch: 0.002815382687930314 - loss: 86.5494 - acc: 0.48:   0%| | 40/42622.979999Epoch: 0.0028857672551285715 - loss: 112.978 - acc: 0.43:   0%| | 41/42622.97999Epoch: 0.002956151822326829 - loss: 108.181 - acc: 0.44:   0%| | 42/42622.979999Epoch: 0.0030265363895250873 - loss: 6.10215 - acc: 0.428571:   0%| | 43/42622.979999999996 [00:51<6:35:38,  1.79it/s][ 405 3283 1323 ..., 1708 1976 2133]
Epoch: 0.003096920956723345 - loss: 92.2868 - acc: 0.56:   0%| | 44/42622.979999Epoch: 0.0031673055239216028 - loss: 84.1623 - acc: 0.51:   0%| | 45/42622.97999Epoch: 0.003237690091119861 - loss: 91.9508 - acc: 0.55:   0%| | 46/42622.979999Epoch: 0.0033080746583181186 - loss: 82.7445 - acc: 0.48:   0%| | 47/42622.97999Epoch: 0.0033784592255163763 - loss: 89.5287 - acc: 0.58:   0%| | 48/42622.97999Epoch: 0.0034488437927146344 - loss: 74.786 - acc: 0.56:   0%| | 49/42622.979999Epoch: 0.003519228359912892 - loss: 81.6576 - acc: 0.55:   0%| | 50/42622.979999Epoch: 0.00358961292711115 - loss: 94.7917 - acc: 0.49:   0%| | 51/42622.9799999Epoch: 0.003659997494309408 - loss: 85.1816 - acc: 0.54:   0%| | 52/42622.979999Epoch: 0.0037303820615076657 - loss: 95.2965 - acc: 0.5:   0%| | 53/42622.979999Epoch: 0.0038007666287059234 - loss: 92.495 - acc: 0.56:   0%| | 54/42622.979999Epoch: 0.003871151195904181 - loss: 92.5675 - acc: 0.57:   0%| | 55/42622.979999Epoch: 0.003941535763102439 - loss: 83.2115 - acc: 0.56:   0%| | 56/42622.979999Epoch: 0.004011920330300697 - loss: 89.4909 - acc: 0.51:   0%| | 57/42622.979999Epoch: 0.004082304897498955 - loss: 99.7805 - acc: 0.44:   0%| | 58/42622.979999Epoch: 0.004152689464697212 - loss: 90.724 - acc: 0.52:   0%| | 59/42622.9799999Epoch: 0.004223074031895471 - loss: 88.3606 - acc: 0.59:   0%| | 60/42622.979999Epoch: 0.004293458599093729 - loss: 84.5603 - acc: 0.57:   0%| | 61/42622.979999Epoch: 0.004363843166291986 - loss: 83.9398 - acc: 0.55:   0%| | 62/42622.979999Epoch: 0.004434227733490244 - loss: 74.6398 - acc: 0.58:   0%| | 63/42622.979999Epoch: 0.004504612300688502 - loss: 74.5205 - acc: 0.63:   0%| | 64/42622.979999Epoch: 0.004574996867886759 - loss: 68.9361 - acc: 0.61:   0%| | 65/42622.979999Epoch: 0.004645381435085017 - loss: 76.3241 - acc: 0.57:   0%| | 66/42622.979999Epoch: 0.004715766002283276 - loss: 86.7402 - acc: 0.52:   0%| | 67/42622.979999Epoch: 0.004786150569481533 - loss: 82.5223 - acc: 0.54:   0%| | 68/42622.979999Epoch: 0.004856535136679791 - loss: 90.1852 - acc: 0.52:   0%| | 69/42622.979999Epoch: 0.004926919703878049 - loss: 78.7655 - acc: 0.61:   0%| | 70/42622.979999Epoch: 0.0049973042710763065 - loss: 88.1767 - acc: 0.53:   0%| | 71/42622.97999Epoch: 0.005067688838274564 - loss: 93.0443 - acc: 0.51:   0%| | 72/42622.979999Epoch: 0.005138073405472823 - loss: 88.6376 - acc: 0.55:   0%| | 73/42622.979999Epoch: 0.0052084579726710805 - loss: 93.3291 - acc: 0.52:   0%| | 74/42622.97999Epoch: 0.005278842539869338 - loss: 82.5663 - acc: 0.61:   0%| | 75/42622.979999Epoch: 0.005349227107067596 - loss: 96.728 - acc: 0.4:   0%| | 76/42622.97999999Epoch: 0.005419611674265854 - loss: 91.9053 - acc: 0.54:   0%| | 77/42622.979999Epoch: 0.005489996241464111 - loss: 77.6307 - acc: 0.52:   0%| | 78/42622.979999Epoch: 0.00556038080866237 - loss: 85.1654 - acc: 0.48:   0%| | 79/42622.9799999Epoch: 0.005630765375860628 - loss: 90.7972 - acc: 0.5:   0%| | 80/42622.9799999Epoch: 0.005701149943058885 - loss: 74.0386 - acc: 0.58:   0%| | 81/42622.979999Epoch: 0.005771534510257143 - loss: 78.7625 - acc: 0.58:   0%| | 82/42622.979999Epoch: 0.005841919077455401 - loss: 91.8244 - acc: 0.51:   0%| | 83/42622.979999Epoch: 0.005912303644653658 - loss: 85.7692 - acc: 0.6:   0%| | 84/42622.9799999Epoch: 0.005982688211851917 - loss: 87.0466 - acc: 0.5:   0%| | 85/42622.9799999Epoch: 0.006053072779050175 - loss: 79.7632 - acc: 0.56:   0%| | 86/42622.979999Epoch: 0.006123457346248432 - loss: 7.98936 - acc: 0.428571:   0%| | 87/42622.979999999996 [01:19<6:26:36,  1.83it/s][ 543  184 2160 ..., 3957 3499  164]
Epoch: 0.00619384191344669 - loss: 64.102 - acc: 0.65:   0%| | 88/42622.97999999Epoch: 0.006264226480644948 - loss: 75.7384 - acc: 0.62:   0%| | 89/42622.979999Epoch: 0.0063346110478432055 - loss: 90.654 - acc: 0.53:   0%| | 90/42622.979999Epoch: 0.006404995615041463 - loss: 79.4898 - acc: 0.6:   0%| | 91/42622.9799999Epoch: 0.006475380182239722 - loss: 86.0056 - acc: 0.52:   0%| | 92/42622.979999Epoch: 0.0065457647494379795 - loss: 75.0733 - acc: 0.6:   0%| | 93/42622.979999Epoch: 0.006616149316636237 - loss: 79.5687 - acc: 0.57:   0%| | 94/42622.979999Epoch: 0.006686533883834495 - loss: 65.1257 - acc: 0.66:   0%| | 95/42622.979999Epoch: 0.006756918451032753 - loss: 83.9232 - acc: 0.55:   0%| | 96/42622.979999Epoch: 0.00682730301823101 - loss: 84.6994 - acc: 0.49:   0%| | 97/42622.9799999Epoch: 0.006897687585429269 - loss: 75.8843 - acc: 0.6:   0%| | 98/42622.9799999Epoch: 0.006968072152627527 - loss: 90.1168 - acc: 0.46:   0%| | 99/42622.979999Epoch: 0.007038456719825784 - loss: 85.5942 - acc: 0.52:   0%| | 100/42622.97999Epoch: 0.007108841287024042 - loss: 65.9521 - acc: 0.68:   0%| | 101/42622.97999Epoch: 0.0071792258542223 - loss: 78.483 - acc: 0.56:   0%| | 102/42622.97999999Epoch: 0.007249610421420557 - loss: 71.3753 - acc: 0.64:   0%| | 103/42622.97999Epoch: 0.007319994988618816 - loss: 74.9723 - acc: 0.59:   0%| | 104/42622.97999Epoch: 0.007390379555817074 - loss: 76.7342 - acc: 0.59:   0%| | 105/42622.97999Epoch: 0.007460764123015331 - loss: 69.8214 - acc: 0.6:   0%| | 106/42622.979999Epoch: 0.007531148690213589 - loss: 83.7998 - acc: 0.54:   0%| | 107/42622.97999Epoch: 0.007601533257411847 - loss: 96.0731 - acc: 0.52:   0%| | 108/42622.97999Epoch: 0.0076719178246101045 - loss: 83.9059 - acc: 0.57:   0%| | 109/42622.9799Epoch: 0.007742302391808362 - loss: 82.4486 - acc: 0.62:   0%| | 110/42622.97999Epoch: 0.00781268695900662 - loss: 89.984 - acc: 0.55:   0%| | 111/42622.9799999Epoch: 0.007883071526204878 - loss: 66.9624 - acc: 0.6:   0%| | 112/42622.979999Epoch: 0.007953456093403135 - loss: 84.125 - acc: 0.62:   0%| | 113/42622.979999Epoch: 0.008023840660601394 - loss: 70.7714 - acc: 0.58:   0%| | 114/42622.97999Epoch: 0.008094225227799652 - loss: 74.6739 - acc: 0.57:   0%| | 115/42622.97999Epoch: 0.00816460979499791 - loss: 86.8767 - acc: 0.58:   0%| | 116/42622.979999Epoch: 0.008234994362196168 - loss: 85.0077 - acc: 0.56:   0%| | 117/42622.97999Epoch: 0.008305378929394425 - loss: 85.5631 - acc: 0.5:   0%| | 118/42622.979999Epoch: 0.008375763496592683 - loss: 79.5271 - acc: 0.52:   0%| | 119/42622.97999Epoch: 0.008446148063790942 - loss: 82.3256 - acc: 0.55:   0%| | 120/42622.97999Epoch: 0.008516532630989199 - loss: 84.2818 - acc: 0.53:   0%| | 121/42622.97999Epoch: 0.008586917198187457 - loss: 75.5408 - acc: 0.64:   0%| | 122/42622.97999Epoch: 0.008657301765385714 - loss: 73.2557 - acc: 0.56:   0%| | 123/42622.97999Epoch: 0.008727686332583973 - loss: 90.7818 - acc: 0.51:   0%| | 124/42622.97999Epoch: 0.00879807089978223 - loss: 76.2838 - acc: 0.63:   0%| | 125/42622.979999Epoch: 0.008868455466980488 - loss: 72.453 - acc: 0.58:   0%| | 126/42622.979999Epoch: 0.008938840034178747 - loss: 81.4822 - acc: 0.56:   0%| | 127/42622.97999Epoch: 0.009009224601377003 - loss: 87.4888 - acc: 0.55:   0%| | 128/42622.97999Epoch: 0.009079609168575262 - loss: 82.3045 - acc: 0.52:   0%| | 129/42622.97999Epoch: 0.009149993735773519 - loss: 74.3538 - acc: 0.65:   0%| | 130/42622.97999Epoch: 0.009220378302971777 - loss: 4.40198 - acc: 0.714286:   0%| | 131/42622.979999999996 [01:47<6:29:52,  1.82it/s]
End of training, validation accuracy: 0.539302, validation loss: 83.6483

**Comment by suri429** [comment]
Hi ,
        how to get the f1 score and precission and recall to our model.?can you please suggest to me.
